pkg/controller/perconaservermongodb/users.go

```diff
@@ -246,7 +246,7 @@ func (r *ReconcilePerconaServerMongoDB) updateSysUsers(ctx context.Context, cr *
 			},
 		}, users...)
 	}
-	if cr.Spec.PMM.Enabled {
+	if cr.Spec.PMM.Enabled && cr.Spec.PMM.HasSecret(newUsersSec) {
```

Comment:
why in line 254 we should put apikey value to nameKey var?

Reply:
It doesn't matter, we are not saving it to the db. We just need to put it somewhere to check if we should create `pmm-client` container